index.html

```diff
@@ -24,7 +24,7 @@
         <script src="./node_modules/react-dom/dist/react-dom.js"></script>
 
         <!-- Main -->
-        <script src="./dist/bundle.js"></script>
+        <script src="./dist/librarie.min.js"></script>
```

Comment:
Is the `min` automatically added to `librarie.js`? And for the production build it will be just `librarie.js`?